### PR TITLE
refactor(example-app): use create action group function for actions

### PR DESCRIPTION
**Pull request by jovanmit998**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

`createAction` function is used for creating actions

Closes #

## What is the new behavior?
Switch to the `createActionGroup` function for a cleaner code in action files
## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/example-app/src/app/auth/actions/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import * as AuthActions from './auth.actions';
-import * as AuthApiActions from './auth-api.actions';
-import * as LoginPageActions from './login-page.actions';
+import { authActions as AuthActions } from './auth.actions';
+import { authApiActions as AuthApiActions } from './auth-api.actions';
+import { loginPageActions as LoginPageActions } from './login-page.actions';
 
 export { AuthActions, AuthApiActions, LoginPageActions };
```
Comment (markostanimirovic) [inline projects/example-app/src/app/auth/actions/index.ts:5]: Let's remove barrel files for actions. With `createActionGroup`, they are not needed anymore.

File: projects/example-app/src/app/books/actions/index.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-import * as FindBookPageActions from './find-book-page.actions';
-import * as SelectedBookPageActions from './selected-book-page.actions';
-import * as ViewBookPageActions from './view-book-page.actions';
+import { bookActions as BookActions } from './book.actions';
+import { booksApiActions as BooksApiActions } from './books-api.actions';
+import { collectionApiActions as CollectionApiActions } from './collection-api.actions';
+import { collectionPageActions as CollectionPageActions } from './collection-page.actions';
+import { findBookPageActions as FindBookPageActions } from './find-book-page.actions';
+import { selectedBookPageActions as SelectedBookPageActions } from './selected-book-page.actions';
+import { viewBookPageActions as ViewBookPageActions } from './view-book-page.actions';
```
Comment (markostanimirovic) [inline projects/example-app/src/app/books/actions/index.ts:7]: same here

File: projects/example-app/src/app/core/actions/index.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import * as LayoutActions from './layout.actions';
-import * as UserActions from './user.actions';
+import { layoutActions as LayoutActions } from './layout.actions';
+import { userActions as UserActions } from './user.actions';
```
Comment (markostanimirovic) [inline projects/example-app/src/app/core/actions/index.ts:2]: here as well

**Review by markostanimirovic — CHANGES_REQUESTED** [review]

**Comment by jovanmit998** [comment]
> Great work! Can you rename action groups to use `PascalCase` instead of `camelCase`? This should greatly reduce the number of changed lines and is also the preferred way of naming action groups in our [official docs](https://ngrx.io/guide/store/action-groups).

Thank you for the review and the comments 🙇 , will switch to `PascalCase`
Just a small question, is there any reason for writing action groups in `PascalCase` instead of `camelCase `?

**Comment by markostanimirovic** [comment]
> Thank you for the review and the comments 🙇 , will switch to `PascalCase` Just a small question, is there any reason for writing action groups in `PascalCase` instead of `camelCase `?

It's just a convention used for NgRx actions for a long time. Camel case is also fine.

Consistent naming across the project is important.

**Review by markostanimirovic — APPROVED** [review]

**Review by timdeschryver — APPROVED** [review]
Thanks @jovanmit998 !

**Pull request merged**